.travis.yml

```diff
@@ -45,14 +47,13 @@ branches:
     - master
     - /^test-.*$/
 
-# container-based infrastructure
-sudo: false
+sudo: required
```

Comment:
That setting was pushed back and forth a lot of times... Why is it necessary this time and how does it relate to what this PR is trying to solve?